messytables/types.py

```diff
@@ -134,6 +155,25 @@ def cast(self, value):
         raise ValueError
 
 
+class TimeType(CellType):
+    result_type = datetime.time
+
+    def cast(self, value):
+        if isinstance(value, self.result_type):
+            return value
+        if value in ('', None):
+            return None
+        hour = int(value[2:4])
+        minute = int(value[5:7])
+        second = int(value[8:10])
+        if hour < 24:
+            return datetime.time(hour, minute, second)
+        else:
+            return datetime.timedelta(hours=hour,
```

Comment:
Here is relevant info regarding TimeDelta: https://github.com/pyexcel/pyexcel-ods3/commit/083b0aa4701bd0997e9fd5638c53758935ca41fb. Basically, ods support time delta where the hour value is greater than or equal to 24.